### ISSUE TITLE
Should the bootstrap css be required in when using ng-bootstrap?

### ISSUE THREAD

**Issue by huntc** [ng-bootstrap/ng-bootstrap#2203]
Being new to this project, the following lines from the "getting started" threw me out completely:

> Should I add bootstrap.js or bootstrap.min.js to my project?
> No, the goal of ng-bootstrap is to completely replace JavaScript implementation for components. Nor should you include other dependencies like jQuery or popper.js. It is not necessary and might interfere with ng-bootstrap code.

I noticed that [the same problem was reported on Stackoverflow](https://stackoverflow.com/questions/48972235/angular-cli-angular-5-2-0-with-ng-boostrap-is-not-working).

Should the getting-started include some words on including the css file within `styles.css` upon having run an `npm install bootstrap`? For example:

```
@import "../node_modules/bootstrap/dist/css/bootstrap.css"
```

**Comment by pkozlowski-opensource** [comment]
YES, you MUST include Boostrap CSS (only CSS, no JS). Could you suggest documentation update to our https://ng-bootstrap.github.io/#/getting-started page?

I would love to have a suggestion for the exact wording you would like to see to make things clear for you and outer users. Thnx!

**Comment by huntc** [comment]
Done: https://github.com/ng-bootstrap/ng-bootstrap/pull/2204

**Comment by huntc** [comment]
(haven't tested how the text renders though)

**Comment by houfu** [comment]
I am thinking of this alternative that provides a bit more hand holding.

**Comment by bastienmoulia** [comment]
Bootstrap should be in peerDependencies of the package.json

And for the import it's better to use the ~ instead of a relative path:
``` css
@import "~bootstrap/dist/css/bootstrap.css"
```

And maybe add an exemple of scss import:
``` scss
@import "~bootstrap/scss/bootstrap"
```

**Comment by dotNetAthlete** [comment]
I think it is bizarre that the css isn't available through ng-bootstrap !

**Comment by arturhungrig** [comment]
I am right now struggling getting the datepicker to be styled the way it is rendered on the Examples page. Absolutely not clear which stylesheet is required.

Not clear whether the default stylesheet should be enough `@import '~bootstrap/dist/css/bootstrap.css';` (in app.component.css for example)

or maybe one should go install `bootstrap-datepicker` and use:
`@import '~bootstrap-datepicker/dist/css/bootstrap-datepicker.css';`

or maybe 
`@import '~bootstrap-datepicker/dist/css/bootstrap-datepicker.standalone.css';`

None seems to work.
Please improve the docs.

**Comment by hcerpandam** [comment]
Hi, i arrived here following the topic https://github.com/ng-bootstrap/ng-bootstrap/pull/2204. 

I've seen that the first time somebody refered to the unclear instructions about the Bootstrap CSS dependency was on feb/10 and still the instructions are really unclear.

As a complete begginer 'im gonna tell you what i've done, in case you want to change something in the getting started section:

**CMD:**

-Creating **new Angular project**

ng new name-of-my-project
Angular routing (y)
Style (css)//¿Can i use scss?

-Installing **Bootstrap**
npm install bootstrap

-Installing **ng-bootstrap**
npm install --save @ng-bootstrap/ng-bootstrap

**IntelliJ:**

In **app-module.ts**

import {NgbModule} from '@ng-bootstrap/ng-bootstrap';
@NgModule({
Imports:[…,NgbModule])}

In **styles.css** or **angular.json**
For **CSS**: @import '~bootstrap/dist/css/bootstrap.css';
For **SCSS**: @import "~bootstrap/scss/bootstrap"

The problem is that i've donde most of the steps reading the comments of people complaining about this issue. Honestly i think that things could be even easier, like using **npm i bootstrap-css-only**, but i don't dare to do any more changes. Learning that took me a couple of days.

Answering the question, i can't see any single reason why Bootstrap CSS shouldn't been included on ng-bootstrap

Thanks

**Comment by sajjad008** [comment]
As a beginner, I found the following the ng-bootrstrap documentation really confusing. I didn't know what was I doing wrong, for the Bootstrap CSS not to take effect. The documentation only talks about not to add Bootstrap JS. It seems I had to add Bootstrap CSS in my index file. There is no harm in mentioning this step in Documentation, if it is a required step.

I guess the Documentation team wants everyone to first read the documentation and then they want (newbies like me) to Google and stumble upon this ticket! Nice way to promote GitHub issue tracking system! :)
